api/src/pcapi/core/external/subcategory_suggestion.py

```diff
@@ -0,0 +1,15 @@
+from pcapi.core.external.subcategory_suggestion_backends import subcategory_suggestion_backend
+from pcapi.core.offerers import api as offerers_api
+
+
+def get_most_probable_subcategory_ids(
+    offer_name: str, offer_description: str | None = None, venue_id: int | None = None
+) -> list[str]:
+    venue = None
+    if venue_id is not None:
+        venue = offerers_api.get_venue_by_id(venue_id)
```

Comment:
```suggestion
    venue = offerers_api.get_venue_by_id(venue_id) if venue_id is not None else None 
```
Ou alors un simple `models.Venue.query.filter(gnagnagna).one_or_none()` ?